docs/for-devs/console/builds.md

```diff
@@ -30,44 +28,43 @@ All Builds of the team are listed here.
 
 ## Create a build
 
-1. Enter a name for the build
+1. Enter a name for the build. The name of the build will be used for the registry name of the image (`harbor.<domainSuffix>/team-name/build-name`).
 
-The name of the build will be used for the registry name of the image (`harbor.<domainSuffix>/team-name/build-name`)
+2. (optional) Adjust the tag. The tag will be used to tag the image  (`harbor.<domainSuffix>/team-name/build-name:tag`).
 
-2. (optional) Adjust the tag
+4. Select the `Repository type`. Select `internal` if you want to use a Git repository hosted in the local Git service (Gitea), or `external` if you want to use an external repository (like one hosted in Github). Select `private` if the external repo requires authentication and select the (`basic-auth` or `ssh-auth`) secret that contains the credentials. Read more about Git authentication [here](https://tekton.dev/docs/how-to-guides/clone-repository/#git-authentication).
```

Comment:
The link to the Tekton docs does not provide an example for the format of the HTTP basic auth Secret. We could also add this reference for that:
https://github.com/tektoncd/catalog/tree/main/task/git-clone/0.9/#using-basic-auth-credentials